compute/instances.go

```diff
@@ -526,18 +536,21 @@ func (c *InstancesClient) qualifyNat(nat []string, shared bool) []string {
 	return qualifiedNats
 }
 
-func (c *InstancesClient) unqualifyNat(nat []string) []string {
+func (c *InstancesClient) unqualifyNat(nat []string) ([]string, error) {
 	unQualifiedNats := []string{}
 	for _, v := range nat {
 		if strings.HasPrefix(v, "ippool:/oracle") {
 			unQualifiedNats = append(unQualifiedNats, v)
 			continue
 		}
 		n := strings.Split(v, ":")
+		if len(n) < 1 {
+			return nil, fmt.Errorf("Error unqualifying nats")
```

Comment:
Since this is in a for loop, we can return the unsplit NAT for further debugging. 
```
return nil, fmt.Errorf("Error unqualifying NAT %s", v)
```